setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = django-oauth-toolkit
-version = 1.2.0
+version = 1.3.0.zapier
```

Comment:
This should be `post1`, not `zapier` per https://www.python.org/dev/peps/pep-0440/#implicit-post-releases.